.travis.yml

```diff
@@ -3,8 +3,12 @@ services: docker
 script: docker build . -t cs50/baseimage
 before_deploy: docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
 deploy:
-  provider: script
+- provider: script
   script: docker push cs50/baseimage
+- provider: script
+  script: docker tag cs50/baseimage cs50/baseimage:ubuntu && docker push cs50/baseimage:ubuntu
```

Comment:
I don't think we want `cs50/baseimage`? We just want `cs50/baseimage:ubuntu` for now? Else building this branch will impact `case/baseimage` and, in turn, `cs50/ecs`?

Reply:
This should just build and push `cs50/baseimage:ubuntu` on push to the ubuntu branch.

Reply:
Oh I see. Why bother tagging it as `cs50/baseimage` then if the tag is unused?

Reply:
Because the same build command is used for `cs50/baseimage:latest`. Alternatively, we could just default the tag name to be the same as the branch name, but we'd have to handle `master` differently (i.e., if current branch is `master`, set tag to `latest`, else set it to branch name).

Reply:
Oh this is just temporary for now, though. Once we merge into master, we'll nix the `:ubuntu` tag altogether.